subworkflows/nf-core/bam_tumor_only_somatic_variant_calling_gatk/meta.yml

```diff
@@ -1,5 +1,5 @@
 # yaml-language-server: $schema=https://raw.githubusercontent.com/nf-core/modules/master/subworkflows/yaml-schema.json
-name: gatk_tumor_only_somatic_variant_calling
+name: bam_tumor_only_somatic_variant_calling_gatk
```

Comment:
will this throw an error in tools or are we catching renamings and handle it automatically in a piplieline?

Reply:
We are not handling name changes in tools, but this is only modifying `meta.yml`, the subworkflow has the correct name `bam_tumor_only_somatic_variant_calling_gatk`.